src/shared/codelens/csharpCodeLensProvider.ts

```diff
@@ -114,29 +117,54 @@ async function onLocalInvokeCommand({
         templatePath: lambdaLocalInvokeParams.samTemplate.fsPath
     })
 
+    // Switch over to the output channel so the user has feedback that we're getting things ready
+    channelLogger.channel.show(true)
+
+    channelLogger.info(
+        'AWS.output.sam.local.start',
+        'Preparing to run {0} locally...',
+        lambdaLocalInvokeParams.handlerName
+    )
+
     try {
         if (!lambdaLocalInvokeParams.isDebug) {
             const baseBuildDir = await makeBuildDir()
             const templateDir = path.dirname(lambdaLocalInvokeParams.samTemplate.fsPath)
             const documentUri = lambdaLocalInvokeParams.document.uri
-            const originalHandlerName = lambdaLocalInvokeParams.handlerName
+            const handlerName = lambdaLocalInvokeParams.handlerName
 
-            const inputTemplate = await makeInputTemplate({
-                baseBuildDir,
-                templateDir,
-                documentUri,
-                originalHandlerName,
-                handlerName: originalHandlerName,
+            const handlerFileRelativePath = getHandlerRelativePath({
+                codeRoot: templateDir,
+                filePath: documentUri.fsPath
+            })
+
+            const relativeFunctionHandler = getRelativeFunctionHandler({
```

Comment:
It's not obvious to me what "relative function handler means". How about calling it something like "full handler name" or "fully-qualified handler name"?

Reply:
The naming convention isn't great but it's what was here before. It's actually a bit of a misnomer too--compiled languages don't have any need for a relative pathing to the handler (if you dig into this function, we return the handler unmodified). 

I'll see what I can do.

Reply:
So on further analysis, when running the Python handler, `args.handlerName` === `relativeOriginalFunctionHandler` and `handlerName` === `relativeFunctionHandler` (everything is equal in the case where we aren't debugging since the debug run creates a separate handler file for the SAM run).

That being said, this logic might potentially change going forward as we improve our code detection logic for the Python handler (right now, we only pull in the code directory itself, so any dependencies in different directories might not get picked up on SAM build). Let's keep this around a little longer and determine whether or not we need it when we improve that.